CHANGELOG.md

```diff
@@ -1,6 +1,22 @@
 # Changelog
 
-All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+All notable changes to this project will be documented in this file. See [commit-and-tag-version](https://github.com/absolute-version/commit-and-tag-version) for commit guidelines.
+
+## [13.0.0](https://github.com/Ninja-Squad/ngx-speculoos/compare/v12.0.1...v13.0.0) (2024-12-06)
+
+
+### ⚠ BREAKING CHANGES
+
+* ngx-speculoos now allows running test in _automatic_ mode. in this mode,
+instead of imperatively triggering a change detection, you await the stability
+and thus let Angular decide if a CD is necessary or not.
+See the "Upgrading to v13" section of the README for details.
```

Comment:
Maybe add a link?

Reply:
It's a mess because the README is in various places.